typescript/infra/config/environments/mainnet3/funding.ts

```diff
@@ -32,6 +32,12 @@ export const keyFunderConfig: KeyFunderConfig<
   contextsAndRolesToFund: {
     [Contexts.Hyperlane]: [Role.Relayer, Role.Kathy],
     [Contexts.ReleaseCandidate]: [Role.Relayer, Role.Kathy],
+    [Contexts.Vanguard0]: [Role.Relayer],
+    [Contexts.Vanguard1]: [Role.Relayer],
+    [Contexts.Vanguard2]: [Role.Relayer],
+    [Contexts.Vanguard3]: [Role.Relayer],
+    [Contexts.Vanguard4]: [Role.Relayer],
+    [Contexts.Vanguard5]: [Role.Relayer],
```

Comment:
ideally use the kf to keep up the balance hourly to the minimum thresholds, and using the script for manual funding where required (especially at the start when we expect a surge)